src/gui/servernotificationhandler.cpp

```diff
@@ -79,9 +79,15 @@ void ServerNotificationHandler::slotNotificationsReceived(const QJsonDocument &j
         a._id = json.value("notification_id").toInt();
         a._subject = json.value("subject").toString();
         a._message = json.value("message").toString();
+
```

Comment:
Ideally we could also use and cache the icon value and use that in the UI, but that is also something that could be done later.